src/OpenApi/src/OpenApiGenerator.cs

```diff
@@ -384,7 +384,6 @@ private List<OpenApiParameter> GetOpenApiParameters(MethodInfo methodInfo, Endpo
             {
                 Name = name,
                 In = parameterLocation,
-                Content = GetOpenApiParameterContent(metadata),
```

Comment:
Is the idea that endpoint `IAcceptsMetadata` should only influence the `RequestBody` and never any `Parameters`?

Adding the same `Content` to all parameters seems bad, but it seems odd that no `OpenApiParameter` ever has any `Content`. It kind of makes sense because there's no `ParameterLocation` for the body and `OpenApiRequestBody` exists, but then what's the purpose of  `OpenApiParameter.Content`?

Reply:
> Edit. I found a sample. It seems like it's if you actually want to do something like accept json in a query string. Weird.

Chatted with @baywet about this. So for 99% of scenarios, the formats that are available for parameters sourced from routes/headers/etc. are going to be described by the `style` and `explode` property (see [this](https://swagger.io/docs/specification/serialization/)).

`Content` is really meant to describe inputs/outputs that are more suited to serialization (request body and response) but is reused here for the 1% of scenarios where people are defining some custom serialization, like the example you shared with queries that look like `?coordiantes={lat:45,long:46}`. For those scenarios, it probably makes sense to have users describe that specification themselves.

Most importantly, it looks like producing this content for the scenarios where it is _not_ needed causes a buggy path in the Swagger UI that causes hangs if you send certain types of inputs through. :/